tests/test_annotations_generator.py

```diff
@@ -0,0 +1,51 @@
+import collections
+
+import pytest
+
+from annotations_ingester.annotations_generator import (
+    AnnotationsMessager,
+    get_uuid_from_graph,
+)
+
+
+@pytest.fixture
+def mock_uuid():
+    return "12345678-1234-1234-1234-123456789012"
+
+
+@pytest.fixture
+def mock_file_contents(mock_uuid):
+    return f"""
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix : <https://test.com> .
+
+:qualityCheckResults {{
```

Comment:
If you use the snippet of RDFLib calls I included in a comment then you'll need to extend this to at least:

```
:qualityCheckResults {
   :checkRun
        a                           eodhqa:EODHQualityMeasurementDataset;
        owl:sameAs                  <urn:uuid:7462319b-947c-4900-83a7-5341362cfab6>.
}

:qualityCheckResults
    a                           dqv:QualityMetadata.
```

Reply:
Extended. It seems to need a lot more